test/tests/meshgenerators/data_driven/tests

```diff
@@ -1,6 +1,6 @@
 [Tests]
   design = 'MeshGenerator.md'
-  issues = '#26295'
+  issues = '#26295 #27075'
```

Comment:
```suggestion
  issues = '#26295'
```

Reply:
27075 rewrites the system, it's appropriate to use both